sql/updates/reverse-dev.sql

```diff
@@ -45,3 +45,5 @@ BEGIN
     WHERE mat_hypertable_id OPERATOR(pg_catalog.=) _cagg_data.mat_hypertable_id;
 END;
 $BODY$;
+
+ALTER TABLE _timescaledb_catalog.dimension DROP COLUMN compress_interval_length;
```

Comment:
Doing catalog table changes like that is problematic and will lead to different schema in new installations vs upgraded installation. You need to rebuild any catalog table you intend to update to prevent that. There are plenty of examples in previous update scripts.